docs/source/user-guide/configuration/pip-interoperability.rst

```diff
@@ -0,0 +1,33 @@
+===================================
+Improving interoperability with pip
+===================================
+
+To improve the interoperability between conda and pip,
+conda 4.6.0 added preview support. With this interoperability,
```

Comment:
`preview support` sounds awkward.  Perhaps:

The conda 4.6.0 release added improved support for interoperability between conda and pip.  This feature is still experimental and is therefore off by default.